system/inc/system_cloud.h

```diff
@@ -44,6 +44,10 @@ typedef enum
 	CLOUD_VAR_BOOLEAN = 1, CLOUD_VAR_INT = 2, CLOUD_VAR_STRING = 4, CLOUD_VAR_DOUBLE = 9
 } Spark_Data_TypeDef;
 
+namespace particle {
+    static const size_t PUBLISH_VITALS_NOW = static_cast<size_t>(-1);
```

Comment:
I think as a general design preference, I would make a typedef that is named after the feature, e.g. `vitalsperiod_t`. This means the code becomes more descriptive and any arguments over what the actual type should be requires just one change :-)

Reply:
This was intended to be a `system_tick_t` and was an oversight. The PRD specifies the name as `particle::NOW`. Both of these changes have been applied.